docs/TestCases.md

```diff
@@ -133,4 +133,112 @@ In the target state, each test case described in the Bash script would be transl
 ![Tech stack](../assets/img/TechStack.png "Tech stack")
 
 ### Inputs
-TODO list inputs for test cases in python
\ No newline at end of file
+TODO list inputs for test cases in python
+
+### Errors and Updates
+
+For Openstack you can check ports and operational tools by using the command:
+`openstack catalog list`
+and will get:
+```
++-------------+----------------+---------------------------------------------------------+
```

Comment:
please remove this table from documentation

Reply:
i understand